lib/metadata_endpoints.js

```diff
@@ -8,11 +8,47 @@ module.exports.smartConfigHandler = async () => {
 		"token_endpoint": 'https://' + process.env.AUTHZ_SERVICE + '/token',
 		"token_endpoint_auth_methods_supported": ["client_secret_basic"],
 		"registration_endpoint": 'https://' + process.env.AUTHZ_SERVICE + '/oidc/register',
-		"scopes_supported": ["openid", "profile", "launch", "launch/patient", "patient/*.*", "user/*.*", "offline_access"],
+		"scopes_supported": [
+			"openid", 
+			"profile", 
+			"launch", 
+			"launch/patient", 
+			"patient/*.*", 
+			"user/*.*", 
+			"offline_access", 
+			'patient/Medication.read', 
+			'patient/AllergyIntolerance.read', 
+			'patient/CarePlan.read', 
+			'patient/CareTeam.read', 
+			'patient/Condition.read', 
+			'patient/Device.read', 
+			'patient/DiagnosticReport.read', 
+			'patient/DocumentReference.read', 
+			'patient/Encounter.read', 
+			'patient/Goal.read', 
+			'patient/Immunization.read', 
+			'patient/Location.read', 
+			'patient/MedicationRequest.read', 
+			'patient/Observation.read', 
+			'patient/Organization.read', 
+			'patient/Patient.read', 
+			'patient/Practitioner.read', 
+			'patient/Procedure.read', 
+			'patient/Provenance.read', 
+			'patient/PractitionerRole.read'
+		],
 		"response_types_supported": ["code", "code id_token", "id_token", "refresh_token"],
 		"introspection_endpoint": '',
 		"revocation_endpoint": 'https://' + process.env.AUTHZ_SERVICE + '/oauth/revoke',
-		"capabilities": ["launch-ehr", "client-public", "client-confidential-symmetric", "context-ehr-patient", "sso-openid-connect"],
+		"capabilities": [
+			"launch-ehr", 
+			"client-public", 
+			"client-confidential-symmetric", 
+			"context-ehr-patient", 
+			"sso-openid-connect", 
+			"context-banner", 
+			"context-style"
```

Comment:
are we actually supporting banner/style? I didn't do anything to support that.

Reply:
in the launch EHR sequence, I append an optional css style link in case that launch is context-banner. 
G10 requires context-banner as a capability, so i've added just added the parameters here to pass.